scrapper/server/db.py

```diff
@@ -0,0 +1,30 @@
+from flask_restful import Api
+from flask_pymongo import PyMongo
+import os
+from pymongo import errors
+
+
+class Db:
+    def __init__(self, app):
+        self.api = Api(app)
+        self.mongodb_client = PyMongo(app, uri=os.environ.get("DB_URI"))
+        self.db = self.mongodb_client.db
+
+    def insert_today_wibor_rates(self, kwargs):
+        try:
+            self.db.wibor_rates.insert_one(kwargs)
+        except errors.ConnectionFailure:
+            print("Could not connect to database")
+
+    def get_todays_wibor(self):
+        try:
+            cursor = self.db.wibor_rates.find().sort([("_id", -1)]).limit(1)
+            result = list(cursor)
+            if len(result) == 0:
+                return
+            wibor_rates = result[0]
+            if wibor_rates == None:
+                return
+            return wibor_rates
```

Comment:
Please refactor this method to work like this (I've not tested it but as far as I remember this should work fine)

```python
def get_todays_wibor(self, date: str):
    for result in self.db.wibor_rates.find({"date": date}):
        return result
```

Reply:
This comment also applies it's related only to database query made with date as a select (date is taken from scrapper result)

Reply:
done